.changeset/wise-jars-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@treasure-dev/tdk-react": patch
```

Comment:
Please mark as a minor change because the `logIn` function signature is changing

Reply:
Done